pom.xml

```diff
@@ -2,15 +2,15 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
-    <artifactId>error-reporting-java</artifactId>
+    <artifactId>error-reporting-java8</artifactId>
```

Comment:
I've seen other libs doing that using the version number. E.g. 0.4.1-jdk8

Reply:
For sql-statement-builder, I added `-java8` suffix. Here I still could add same, `error-reporting-java-java8`. 
What do you think? I think it is still okay, since we are going to keep the branch for limited time.

Reply:
ok, just keep it like it is

Reply:
For you suggestion, I will have to change the final artifact name. Since I would like to keep the version schema same with main branch.